irohad/multi_sig_transactions/state/impl/mst_state.cpp

```diff
@@ -28,9 +31,15 @@ namespace iroha {
                        });
   }
 
-  bool DefaultCompleter::operator()(const DataType &tx,
+  bool DefaultCompleter::operator()(const DataType &batch,
                                     const TimeType &time) const {
-    return false;
+    return std::any_of(batch->transactions().begin(),
+                       batch->transactions().end(),
+                       [&](const auto &tx) {
+                         return tx->createdTime()
+                             + expiration_time_ / std::chrono::milliseconds(1)
+                             < time;
```

Comment:
That is not a problem of your PR, but it is not evident, which time scale is used in the `TimeType`. @muratovv, should we place a TODO to make such conversions in a standard way?